05_stringer/sahidakhtar1/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+type MyByte byte
```

Comment:
I probably should have been more clear in my previous comment. The issue is if you add a type like this with it's own stringer, it can mess up the behaviour of your own stringer implementation using `%v`. Using `%d` protects you from that in the future.
You don't need `MyByte`, the code I gave you is just meant to illustrate the danger of using `%v` for what is expected to be literals.

Reply:
Got it. If define type `MyByte` with its own stringer and use %v then it will call `MyByte`'s stringer instead of `IPAddr` stringer. Right?

Reply:
Almost, `%v` will look at the `MyByte` stringer, whereas `%d` only cares about the underlying `byte`.